src/pages/IDResolutionPage/IDResolveRedirectionPage.tsx

```diff
@@ -0,0 +1,104 @@
+import { CSSProperties } from 'react';
+import { useQuery } from 'react-query';
+import { useSelector } from 'react-redux';
+import { useHistory, useParams } from 'react-router';
+import { useNexusContext } from '@bbp/react-nexus';
+import { Button, Modal } from 'antd';
+import { isEmpty } from 'lodash';
+
+import { RootState } from '../../shared/store/reducers';
+import { ResourceJSONPrettify } from './IDResolvedManyPage';
+
+const modalStyle = {
+  display: 'flex',
+  alignItems: 'flex-start',
+  justifyContent: 'center',
+  flexDirection: 'column',
+} as CSSProperties;
+
+const calloutStyle = {
+  margin: 10,
+  width: '100%',
+  border: '1px solid #c11b1b3b',
+  padding: '4px',
+  borderRadius: '4px',
+  fontSize: 14,
+  fontWeight: 'bold',
+} as CSSProperties;
+
+const IDResolveRedirectionPage = () => {
+  const nexus = useNexusContext();
+  const oidc = useSelector((state: RootState) => state.oidc);
```

Comment:
I think it may make sense to have the selector directly return if the user is authenticated or not. (Maybe we can even have that as a global selector if possible). Something like this:

```typescript
const isAuthenticated = useSelector((state) => {
    const oidc = state.oidc;
    const token = oidc && oidc.user && !!oidc.user.access_token;
    return !isEmpty(oidc.user) && token;
});
```
What do you think?

Reply:
we can do that, but we should `memoize` the `selector fn`